glean-core/metrics.yaml

```diff
@@ -81,6 +81,7 @@ glean.internal.metrics:
       - glean_client_info
     description: |
       The manufacturer of the device the application is running on.
+      Not set if the device manufacturer can't be determined (e.g. on Desktop).
```

Comment:
pebble: It's not that it *can't*, it's that it's not terribly useful or accurate.

Reply:
He, I left that vague enough in case we ever reconsider it :D

But I'm happy to change it to some more appropriate.